### DISCUSSION TITLE
How are branch subdomains supported with translations?

### DISCUSSION THREAD

**Discussion by tony19** [tony19/vite-docs-template#6] [Q&A] [answered]
For instance, there is https://v2.vitejs.dev, but what is the equivalent for the Spanish translation? It's apparently not https://v2.es.vitejs.dev.

@mrcego

**Comment by mrcego** [comment, marked as answer]
Current translation was originally intended for v2, but me and @patak-dev decided spanish translation was available from v3, if I understand your question. So there is not Spanish docs for v2.

**Reply by tony19** [reply]
Ok. But the question still stands for the next major version of Vite, assuming there will still be Spanish translations of the docs.

**Reply by mrcego** [reply]
I think is http://es.vitejs.dev/v3/ following https://es.vuejs.org/v2/ example. Thoughts?

**Reply by tony19** [reply]
~Hmm. I personally prefer the URL scheme you're suggesting for aesthetics, but it seems inconsistent because the version would no longer be in the subdomain. Perhaps we should combine them into one: https://v2-es.vitejs.dev. It's not pretty though.~

After some thought, I ultimately like your suggested URL scheme best despite a minor inconsistency. It's a worthwhile tradeoff IMO.
